plugins/hypervisors/kvm/src/main/java/com/cloud/hypervisor/kvm/resource/LibvirtComputingResource.java

```diff
@@ -3459,6 +3457,8 @@ public boolean isCentosHost() {
                 platformEmulator.startsWith("Oracle") ||
                 platformEmulator.startsWith("Other PV")) {
             return DiskDef.DiskBus.VIRTIO;
+        } else if (isUefiEnabled && (platformEmulator.startsWith("Windows") || platformEmulator.startsWith("Other"))) {
```

Comment:
To simplify those validations, we could use `StringUtils.startsWithAny`, from `commons.lang3`:

```java
StringUtils.startsWithAny(platformEmulator, ...
```

Reply:
@GutoVeronezi, I saw your suggestion, and I like it, but I'm not sure that I have to do it. The first is because I was following the rest of the statement to be in the "same style", and it's a small fix, not a refactoring. Another thing is in `LibvirtComputingResource` is used `com.cloud.utils.StringUtils` and `org.apache.commons.lang.StringUtils`, for me, it's going to be a mess if we use and `commons.lang3`. 
Anyway, I will make the change :)